invenio_oauthclient/views/settings.py

```diff
@@ -41,14 +41,23 @@
 )
 
 
+def menu_item_visible():
+    """Menuitem should't be displayed if no providers have been configured."""
+    if not current_app.config['OAUTHCLIENT_REMOTE_APPS']:
+        return False
+    else:
+        return True
+
+
 @blueprint.route('/', methods=['GET', 'POST'])
 @login_required
 @register_menu(
     blueprint, 'settings.oauthclient',
     _('%(icon)s Linked accounts', icon='<i class="fa fa-link fa-fw"></i>'),
     order=3,
     active_when=lambda: request.endpoint.startswith(
-        'invenio_oauthclient_settings.')
+        'invenio_oauthclient_settings.'),
+    visible_when=menu_item_visible
```

Comment:
You are right, this is more in line to the previously used coding style. Changed to this one. Thanks.